recipes/hipify/recipe.yaml

```diff
@@ -0,0 +1,52 @@
+context:
+  name: hipify
+  version: "7.0.2"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/ROCm/HIPIFY/archive/refs/tags/rocm-${{ version }}.tar.gz
+  sha256: d6e78b025c2cb36f9470d1ec572adc8abc7f2c79bb9a5e21cf46fabd305c4b9c
+  patches:
+    - fix-llvm21-diagnosticoptions.patch
+
+build:
+  number: 0
+  skip:
+    - not linux
+
+requirements:
+  build:
+    - ${{ compiler('cxx') }}
+    - cmake
+    - ninja
+  host:
+    - llvmdev
```

Comment:
Please add `rocm-core` to ensure that we do not mix different versions of hipify with different versions of rocm, including incompatible version of clang.